dspace/config/launcher.xml

```diff
@@ -242,6 +242,13 @@
 	    <argument>reindex</argument>
         </step>
     </command>
+    <command>
+        <name>solr-upgradeD6-statistics</name>
```

Comment:
Silly thing, but I wonder if we should name this something more obvious like `solr-upgrade-statistics-v6`?   The embedded `D6` is not as obvious to me, but maybe it is for others?  Plus, I'm assuming we may need future `solr-upgrade-statistics` scripts (at some point), and it might be easier to sort them out if the version is on the end.

Reply:
Given that, in the foreseeable future, there may be sites upgrading right over v6 straight to v7, maybe these names should be about what we are upgrading *from*.  So, "solr-upgrade-pre6-statistics" or some such.